app/src/ui/branches/branch-list-item.tsx

```diff
@@ -24,17 +24,20 @@ export class BranchListItem extends React.Component<IBranchListItemProps, {}> {
     const lastCommitDate = this.props.lastCommitDate
     const isCurrentBranch = this.props.isCurrentBranch
     const name = this.props.name
+    const hasMatches = this.props.matches.length > 0
+    const matches = hasMatches ? this.props.matches[0] : []
```

Comment:
We're doing this in a couple of places - could this be extracted to a function like `getFirstMatchesOrEmpty`?

```ts
function getFirstMatchesOrDefault(matches: ReadonlyArray<ReadonlyArray<number>>) {
  returns matches.length > 0 ? matches[0] : []
}

...

const matches = getFirstMatchesOrDefault(this.props.matches)
```

Reply:
I think it's appropriate to keep this in `lib/fuzzy-find.ts` alongside the other logic for this.